queue_services/payment-reconciliations/src/reconciliations/payment_reconciliations.py

```diff
@@ -755,6 +756,9 @@ def _create_nsf_invoice(cfs_account: CfsAccountModel, inv_number: str,
     )
     inv_ref.save()
 
+    # Add invoice to non sufficient funds table
+    NonSufficientFundsService.save_non_sufficient_funds(invoice_id=invoice.id, description='NSF')
```

Comment:
I'm not sure if this going to capture all of the invoices that are NSF - this will only capture the one returning back, I don't know if CAS will return rows for all of the NSF invoices.